internal/sdk/cloudian/sdk.go

```diff
@@ -291,7 +301,7 @@ func (client Client) ListUserCredentials(ctx context.Context, user User) ([]Secu
 		return securityInfo, nil
 	case 204:
 		// Cloudian-API returns 204 if no security credentials found
-		return nil, ErrNotFound
+		return nil, nil
```

Comment:
Why are we changing this?

Reply:
Because it's more intuitive to get en empty list than a no-elements-found error. The error should imo instead be returned when the user we request keys of does not exist.

Reply:
I agree, but can we make this change in a dedicated PR? It is not directly related to the desired goal here.